guides/common/modules/proc_verifying-dns-resolution.adoc

```diff
@@ -42,10 +42,6 @@ rtt min/avg/max/mdev = 0.019/0.019/0.019/0.000 ms
 # hostnamectl set-hostname _name_
 ----
 
-ifdef::satellite[]
-For more information, see the https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html-single/networking_guide/index#sec_Configuring_Host_Names_Using_hostnamectl[Configuring Host Names Using hostnamectl] in the _{RHEL} 7 Networking Guide_.
```

Comment:
Is there no RHEL 8 equivalent?

Reply:
I have raised the query and the link to EL8 should be obtained soon.

Reply:
Added the new link for EL8. Addressed :+1: